### PR TITLE
Improved documentation for edX OIDC configuration

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-2345

### CODE REVIEW

**Comment by clintonb** [comment]
@rlucioni @schenedx @dsjen @bderusha please review. These instructions, specifically the view/url configuration, can be used to update your IDAs with single logout support.

**Comment by clintonb** [comment]
A rendered version is available at https://github.com/edx/auth-backends/tree/clintonb/sso-docs.

File: README.rst [hunk 1]
```diff
@@ -6,10 +6,8 @@ auth-backends  |Travis|_ |Codecov|_
 .. |Codecov| image:: http://codecov.io/github/edx/auth-backends/coverage.svg?branch=master
 .. _Codecov: http://codecov.io/github/edx/auth-backends?branch=master
 
-This repo houses custom authentication backends, views, and pipeline steps used by edX
-projects such as the `edx-analytics-dashboard <https://github.com/edx/edx-analytics-dashboard>`_
-and `ecommerce <https://github.com/edx/ecommerce>`_. These components are primarily intended to aid the usage
-of edX's single sign-on functionality (built atop OpenID Connect).
+This repo houses a custom authentication backend, views, and pipeline steps used by edX services for single sign-on.
+This functionality is built atop the `OpenID Connect protocol <http://openid.net/connect/>`_
```
Comment (rlucioni) [inline README.rst]: Nitpick: missing period.

File: README.rst [hunk 2]
```diff
@@ -20,49 +18,83 @@ The `auth_backends` package can be installed from PyPI using pip::
...
     $ pip install edx-auth-backends
 
-Backends
---------
+Configuration
+-------------
+Adding single sign-on/out support to a service requires a few changes:
 
-Included backends:
+1. Define settings.
```
Comment (rlucioni) [inline README.rst]: Nitpick: unnecessary period. :)

File: README.rst [hunk 3]
```diff
@@ -20,49 +18,83 @@ The `auth_backends` package can be installed from PyPI using pip::
...
+        'django.contrib.auth.backends.ModelBackend',
+    )
+
+Authentication Views
+~~~~~~~~~~~~~~~~~~~~
+In order to make use of the authentication backend, your service's login/logout views need to be updated. The login
+view should be updated to redirect to the OpenID Connect provider's login page. The logout view should be updated to
+redirect to the OpenID Connect provider's logout page.
+
+This package includes views and urlpatterns that configured for edX OpenID Connect. To use them, simply append/prepend
```
Comment (rlucioni) [inline README.rst]: views and urlpatterns that *are* configured

**Comment by rlucioni** [comment]
👍 just some minor comments.

File: README.rst [hunk 4]
```diff
@@ -20,49 +18,83 @@ The `auth_backends` package can be installed from PyPI using pip::
...
+    EXTRA_SCOPE = ['preferred_language']
 
-urls.py
--------
+Assuming the identity provider knows how to process that scope, the associated claim data will be included in the
+identity token returned during authentication. Note that these scopes/claims are dependent on the identity provider
+being used. The ``EdXOpenIdConnect`` backend is configured to be used by all edX services out-of-the-box.
+
+There is an additional setting, ``COURSE_PERMISSIONS_CLAIMS``, used by
+`edx-analytics-dashboard <https://github.com/edx/edx-analytics-dashboard>`_. Its usage is not recommended due to
```
Comment (dsjen) [inline README.rst]: I'm not sure this is totally accurate.  The initial performance concerns were in querying courses that the user had access to and was slow prior to using the course overviews table to provide the list of courses.

**Pull request merged**